e2e/tests/query.feature

```diff
@@ -0,0 +1,341 @@
+
+  Scenario: To query an address balance in the network
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query balance {{address}} 0" against validator 0
+	Then the user should be able to see standard output containing "/v1/query/balance"
+
+  Scenario: To query an address balance in the network, wrong address
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query balance {{address}} 0" against validator -1
+	Then the user should be able to see standard output containing "invalid byte"
+
+  Scenario: To query an address balance in the network, wrong height
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query balance {{address}} -3" against validator 0
+	Then the user should be able to see standard error containing "unknown shorthand flag: '3' in -3"
+
+  Scenario: To query an address balance in the network, incomplete command
+	Given the user has a pocket client
+	When the user runs the command "query balance"
+	Then the user should be able to see standard error containing "Usage:"
+
+  Scenario: To query a block at height, wrong height
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query block 0"
+	Then the user should be able to see standard output containing "height must be greater than 0"
+
+  Scenario: To query a block's at height, given an invalid number height
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query block 1000"
+	Then the user should be able to see standard output containing "must be less than or equal to the current blockchain height"
+
+  Scenario: To query a block's at height, given an invalid char in height
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query block 0-0"
+	Then the user should be able to see standard output containing "invalid syntax"
+
+  Scenario: To query block-txs at height
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query block-txs 0 1 1 1"
+	Then the user should be able to see standard output containing "/v1/query/blocktxs"
+	And the user should be able to see standard output containing "total_txs"
+
+  Scenario: To query a block-txs at height, incomplete
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query block-txs"
+	Then the user should be able to see standard error containing "Usage:"
+
+  Scenario: To query dao owner
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
```

Comment:
Let's try to use one Given statement per scenario. We can combine both statements into one Given statement for future usage but ideal to have one Given statement.